Update support page (#30243)
dotnet/docs: docs/core/releases-and-support.md

---
title: .NET releases, patches, and support
description: Learn about releases, patches, and support for .NET 5+ (including .NET Core) and later versions.
ms.date: 7/15/2022
ms.topic: overview
ms.author: tdykstra
author: tdykstra
---
# Releases and support for .NET

This article describes release types, servicing updates, SDK feature bands, support periods, and support options for .NET.

## Release cadence

New major .NET versions are released annually in November, at [.NET Conf](https://www.dotnetconf.net/).

Patch updates are released monthly on the second Tuesday of each month, also known as Patch Tuesday.

Minor versions of the .NET SDK are released approximately quarterly. These are known as [feature bands](#feature-bands-sdk-only).

## Release types

Each .NET release is defined as either  **Short Term Support (STS)** or **Long Term Support (LTS)**, at the beginning of the release.

> The release types:

Reviewer: I think in here we shouldn't introduce the release type names ahead of their definition. Something like this would look better:
![image](https://user-images.githubusercontent.com/12971179/182269376-28ac616a-017e-4f00-9e59-fc0d511f4add.png)

Then you can rewrite the intro to not mention the names.

## Support phases

.NET releases go through multiple support phases, with varying support levels.

* **Preview** releases are not supported but are offered for public testing and for the opportunity to give feedback.
* **Go-live** support enables users to deploy a pre-release build in production, and are supported by Microsoft. These are typically Release Candidate (RC) releases.
* **Active** support is provided for the majority of the period after a release is Generally Available (GA). Functional and security improvements will be provided, including support for new operating system versions.
* **Maintenance** support is provided for the last six months of support. Improvements are limited to security fixes. Support for new operating system versions will be provided on a best-effort basis.
* **End-of-life (EOL)** marks the point where a release is no longer supported.

.NET uses the [modern lifecycle](/lifecycle/policies/modern) for support and servicing. It is also used for .NET Framework.

[Support for various operating systems](https://github.com/dotnet/core/blob/main/os-lifecycle-policy.md) is defined for each release.

> ## Product behavior

Reviewer 2: ```suggestion
## .NET runtime and SDK versions
```

> Some .NET behaviors are affected by product version.

Reviewer 2: ```suggestion
Some .NET releases are only for the runtime or only for the SDK.
```

> ### Runtime roll-forward and compatibility

Reviewer 2: ```suggestion
### Runtime versions
```

All .NET updates install side-by-side with previous versions. An application built to target a specific *major.minor* version continues to use that targeted runtime even if a newer version is installed. The app doesn't automatically roll forward to use a newer *major.minor* version of the runtime unless you opt in for this behavior. An application that was built to target .NET 6 won't automatically start running on .NET 7.

We recommend rebuilding apps and testing them with a newer major runtime version before deploying to production. For more information, see [Framework-dependent apps roll forward](versions/selection.md#framework-dependent-apps-roll-forward) and [Self-contained deployment runtime roll forward](deploying/runtime-patch-selection.md).

Servicing updates are treated differently from major releases. An application built to target .NET 6.0.0 will automatically roll-forward to use a newer patch -- like 6.0.1 -- when that servicing update is installed.

> ## Feature bands (SDK only)

Author: ```suggestion
### Feature bands (SDK only)
```

### Community support

> For community support, see [dotnet/core](https://github.com/dotnet/core) or our [Community page](https://dotnet.microsoft.com/platform/community).

Reviewer 2: ```suggestion
For community support, open an issue in the [dotnet/core](https://github.com/dotnet/core) GitHub repo or on our [Community page](https://dotnet.microsoft.com/platform/community).
```
